### ISSUE TITLE
Image preprocessing before inference?

### ISSUE THREAD

**Issue by MalharJ** [MadryLab/robustness#74]
For normal PyTorch models we'd usually perform some sort of image preprocessing before inference (https://pytorch.org/docs/stable/torchvision/models.html), with a mean and a standard deviation. Do we do the same for the robust pre-trained models in this library?

**Comment by andrewilyas** [comment]
Yep! Check out the ``AttackerModel`` class, normalization is implemented there.

**Comment by andrewilyas** [comment]
Closing this now, let me know if you have any questions!
